### ISSUE TITLE
Please tag a release

### ISSUE THREAD

**Issue by dpryan79** [lbcb-sci/graphmap2#6]
Could you tag a release? We can then get the updated version of this in bioconda.

**Comment by drpatelh** [comment]
Yes, that would be great. We would like to use the BioContainer for `graphmap2` in a pipeline we are developing for Nanopore sequencing analysis:
https://github.com/nf-core/nanoseq

Currently, we are using the `graphmap` BioContainer as a placeholder until this is added:
https://github.com/nf-core/nanoseq/blob/809d0a74f1afda0c6904f09b22eb7d3717aff740/conf/base.config#L84

**Comment by jmaricb** [comment]
I have added the release version v0.6.3. Let me know if that is what you need and if you need something more from me.

**Comment by dpryan79** [comment]
Awesome, that solves things for us!
